rootfs/usr/share/inputplumber/devices/60-ps5_ds_gamepad.yaml

```diff
@@ -17,29 +17,30 @@ single_source: false
 matches: []
 
 source_devices:
-  # Read events from the evdev device
   - group: gamepad
+    hidraw:
+      vendor_id: 0x054c
+      product_id: 0x0ce6
+
+  # Block the evdev implementations
+  - group: gamepad
+    blocked: true
     evdev:
-      name: Sony Interactive Entertainment DualSense Wireless Controller
+      name: "{Sony Interactive Entertainment DualSense Wireless Controller,DualSense Wireless Controller}"
```

Comment:
While it might not matter, this is going to attach any additional evdevs from additional controllers to the first controller. Given that there is *some* processing that occurs for each event, even if blocked, this could negatively impact latency on the first controller compared to the others.

Reply:
Blocked source devices are only hidden and never run. So there will be no event processing for blocked devices.

Reply:
Also, these are set to `unique: true`, so only one will be added to each composite device.